### ISSUE TITLE
Fails when module and class has same name

### ISSUE THREAD

**Issue by jepperaskdk** [jepperaskdk/pydoctest#34]
This example fails since `datetime` is both the name of the class and the module.

```
from datetime import datetime


def test(date: datetime) -> None:
    """
    This should work.

    Args:
        date (datetime): The date.
    """
    return None

# Argument type differ. Argument 'date' was expected (from signature) to have type '<class 'datetime.datetime'>', but has (in docs) type '<module 'datetime' from '/usr/lib/python3.10/datetime.py'>'
```

**Comment by Risitop** [comment]
Indeed, encountered this issue with the `tqdm` module: `from tqdm import tqdm`.

Any idea on how to patch/ignore this issue?

**Comment by jepperaskdk** [comment]
@Risitop Thanks for the comments. I have made a PR here if you want to review: https://github.com/jepperaskdk/pydoctest/pull/56

**Comment by jepperaskdk** [comment]
@Risitop This is now fixed in [0.2.1](https://github.com/jepperaskdk/pydoctest/releases/tag/0.2.1)
